src/main/scripts/install.sh

```diff
@@ -225,6 +225,10 @@ copy_jdbc_drivers() {
 
     if [ $dbType == "db2" ]; then
         find ${WAS_ND_INSTALL_DIRECTORY} -name "db2jcc*.jar" | xargs -I{} cp {} "$jdbcDriverPath"
+    elif [ $dbType == "oracle" ]; then
+        # Download jdbc drivers
+        curl -Lo ${jdbcDriverPath}/ojdbc8.jar https://download.oracle.com/otn-pub/otn_software/jdbc/1916/ojdbc8.jar
```

Comment:
Sorry to have to ask for this, but please put this inside a retry loop.

Reply:
Good point. Fixed in [af2fa69](https://github.com/WASdev/azure.websphere-traditional.cluster/pull/184/commits/af2fa69428762dc497f0adb3d93323d3a4774e9d).